SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/UtilsShared/DataBrokerProtectionSettings.swift

```diff
@@ -93,6 +93,11 @@ public final class DataBrokerProtectionSettings {
         lastBrokerJSONUpdateCheckTimestamp = timestamp ?? Date().timeIntervalSince1970
     }
 
+    public func resetBrokerDeliveryData() {
+        mainConfigETag = nil
+        updateLastSuccessfulBrokerJSONUpdateCheckTimestamp(Date.distantPast.timeIntervalSince1970)
+    }
```

Comment:
This fixes an issue with remote broker delivery where deleting a profile data doesn’t delete the stored main config ETag, leading to stale data.